libs/core/tests/unit_tests/runnables/test_runnable.py

```diff
@@ -1020,6 +1020,115 @@ def test_configurable_alts_factory() -> None:
     assert fake_llm.with_config(configurable={"llm": "chat"}).invoke("...") == "b"
 
 
+def test_configurable_fields_prefix_keys() -> None:
+    fake_chat = FakeListChatModel(responses=["b"]).configurable_fields(
+        responses=ConfigurableFieldMultiOption(
+            id="responses",
+            name="Chat Responses",
+            options={
+                "hello": "A good morning to you!",
+                "bye": "See you later!",
+                "helpful": "How can I help you?",
+            },
+            default=["hello", "bye"],
+        ),
+        sleep=ConfigurableField(
+            id="chat_sleep",
```

Comment:
Was surprising to find `sleep` in here, didn't realize it was an actual attribute on fake list chat model (it doesn't sound like it would be, but it is), maybe a comment could be helpful